src/App.js

```diff
@@ -1,24 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator.js';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator text=""/>
```

Comment:
- [ ] It is required that all the components in the app should be class-based component. Kindly change this component to class-based component in order to meet with project requirements.

![project-reqtmt](https://user-images.githubusercontent.com/26597432/206406370-e82a7b0b-107c-47cd-91fb-3866154213d7.png)